diagnostic.md

```diff
@@ -50,7 +49,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
+Relative path because it is locally sourced.
```

Comment:
This is an absolute path.  The leading forward slash (/) indicates the root of the system.